Adobe/CFBundleVersionExtensionAttribute.xml

```diff
@@ -0,0 +1,19 @@
+<computer_extension_attribute>
+	<name>%NAME%Version</name>
+	<description />
+	<data_type>String</data_type>
+	<input_type>
+		<type>script</type>
+		<platform>Mac</platform>
+		<script>#!/bin/assh
```

Comment:
Those will actually not be needed, but the current were complaining about missing the .xml.. again.. bare with.. will tidy